### PR TITLE
Website Overhaul

### PR DESCRIPTION
**Pull request by ShaunR1991**
# Description

- Updated splash page:
 - Fixed broken links
 - Adjusted the About Us language
 - Updated quick links to more popular pages:
	1. Site Introduction
	2. Onboarding Hub (content WIP)
	3. Documentation
	4. News
	
- Content pages restructured to group like pages for ease of navigation. 

- Updates "Processes" tab
 - Fixed all broken links
 - Overall improved language to be more concise, gramattically correct, free of spelling errors and correct punctuation.
 - Renamed to "Resources" to be more appropriate for content
 - Updated language in the introduction 
 - Validated and expanded on the technology stack documentation
 - Removed "Cyber Security Guidelines" parent folder to have the "Remote Working Guide" unnested.
 - Created Onboarding Hub
  - Moved the Thoth Tech Welcome Package to here from "Policies"
  - Created placeholders for current teams
 - Updated Quality Assurance Overview document language 
 - Validated and updated the Git Contributions Guide. 
  - Added explanations to commands
  - Expanded some explanations
  - Improved overall language
 - Updated Testing and Development content
 - Updated Documentation Tab
  - Documenting for Thoth Tech document format adjusted and languate improved.
  - Writing Style Guide formatting improved; examples provided for all items. 
  - Templates moved to Documentation tab for coherence
 - FAQs moved to Resources; added a few additional FAQs. 
 
- Consolidated Teams and Leadership page into trimesters.
- Included current trimester information into Teams and Leadership page.

- Consolidated Company Submissions into appropriate trimesters for ease of navigation.

## Type of change

_Please delete options that are not relevant._

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation (update or new)

## How Has This Been Tested?

Tested in Astro dev environment with no new errors. 
Tested that all links work.

## Testing Checklist

- [x] Tested in latest Chrome
- [x] Tested in latest Safari
- [x] Tested in latest Firefox

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have requested a review from SplashKit Leadership Team on the Pull Request

### CODE REVIEW

**Comment by breezy-codes** [comment]
This is great, the site looks much better with the new organisation and the additions of the new team lists for this semester.

**Review by breezy-codes — APPROVED** [review]
This is great, the site looks much better with the new organisation and the additions of the new team lists for this semester.

File: src/content/docs/resources/faq.mdx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+Yes, there is. Find some helpful links below:
+
+- [Instructional video](https://deakin365.sharepoint.com/:v:/r/sites/ThothTech2/Shared%20Documents/General/Instructional%20Video%20-%20How%20To%20Upload%20Documents%20to%20Starlight.mp4?csf=1&web=1&e=s26lMo).
+- [Help hub session recording](https://deakin365-my.sharepoint.com/:v:/g/personal/s221460489_deakin_edu_au/EXib27xJI2hCkHpzBXOxxYQBOfDGhhQJ1A23ciisMROeRA?xsdata=MDV8MDF8fDgxMTZlNDYzMzAyZTQxYTZhNGE2MDhkYmYyMWMwMGY1fGQwMjM3OGVjMTY4ODQ2ZDU4NTQwMWMyOGI1ZjQ3MGY2fDB8MHw2MzgzNjk5NzQyOTIxNjE3MDJ8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxMM1JsWVcxekx6RTVPbEZtZUY5VFZFaFZPVEJQYzFaWlFraFdXVXRvYzFKUk5XZHRSV1V3Y3psUk5tdFBjRUptTm1Kc2FUZ3hRSFJvY21WaFpDNTBZV04yTWk5amFHRnVibVZzY3k4eE9UcFJabmhmVTFSSVZUa3dUM05XV1VKSVZsbExhSE5TVVRWbmJVVmxNSE01VVRaclQzQkNaalppYkdrNE1VQjBhSEpsWVdRdWRHRmpkakl2YldWemMyRm5aWE12TVRjd01UUXdNRFl5T0RNd01nPT18NTg0ZjIwNWZmZjA3NGZlZTQyNjQwOGRiZjIxYzAwZjN8ZTU2N2Y4NzA4YjFlNDI1YWIzYjRhM2I1NmExY2QwMjE%3D&sdata=TldtNGVnTDJWWlVnb2lBMGlZZXlLNEI1M2p5emU3ZndBYmpSTkV6MnN0QT0%3D).
```
Comment (Liquidscroll) [inline src/content/docs/resources/faq.mdx:48]: I don't have access to this video, is this just a permission issue for me or can anyone else view this video?

File: src/content/docs/teams-and-leadership/2024 Trimester 2/leadership-t2-2024.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+- OnTrack: Nebula Alam & Satika Jayawardena
+- SplashKit: Olivia McKeon, Sean Boettger & Tien Pham
+
+### Student Leadership
+
+- Jake Mansfield - Student Lead  | OnTrack CourseFlow Development
+- Jack Carroll - Student Lead  | OnTrack JPLag Implementation
+- Atharv Bhandare - Junior Leader  | OnTrack JPLag Implementation
+- Isabelle Holt - Senior Leader  | OnTrack FrontEnd Migration
```
Comment (Liquidscroll) [inline src/content/docs/teams-and-leadership/2024 Trimester 2/leadership-t2-2024.md:19]: AFAIK Isabelle Holt isn't a leader anymore

File: src/content/docs/teams-and-leadership/2024 Trimester 2/team-members-t2-2024.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+| Nidhisha Pahade      | Team Member – SplashKit Website                  | Senior        |
+| William Saunders     | Team Member – Arcade Machine & Expansion         | Senior        |
+| Thomas Shanahan      | Team Member – Games Development                  | Senior        |
+| Darren Sunandar      | Team Member – SplashKit Expansion                | Senior        |
+| Taiming Xu           | Team Member – SplashKit Website                  | Senior        |
+| Yuyang Yang          | Team Member – SplashKit Tutorials                | Senior        |
+| Simon Zhu            | Team Member – Games Development                  | Senior        |
+| Brianna Laird        | Junior Lead – SplashKit Tutorials & Expansion    | Junior        |
+| Shaun Ratcliff       | Junior Lead – SplashKit Website & Tutorials      | Junior        |
+ Jordan Brown         | Team Member – SplashKit Website                  | Junior        |
```
Comment (Liquidscroll) [inline src/content/docs/teams-and-leadership/2024 Trimester 2/team-members-t2-2024.md:54]: Missing |

**Review by Liquidscroll — CHANGES_REQUESTED** [review]
Great work! I really like how you've organised it, it's definitely given the website a professional refresh.

There's a few small changes to be made.

The other issue with this, however, is the same as in your other PR (https://github.com/thoth-tech/ThothTech-Documentation-Website/pull/87). They both include the same changes and will conflict when we merge these in. This will need to be fixed.

File: src/content/docs/resources/faq.mdx (hunk 1 above)
Comment (omckeon) [reply src/content/docs/resources/faq.mdx:48]: I can't access that video either

File: src/content/docs/teams-and-leadership/2024 Trimester 2/team-members-t2-2024.md (hunk 3 above)
Comment (ShaunR1991) [reply src/content/docs/teams-and-leadership/2024 Trimester 2/team-members-t2-2024.md]: Fixed :)

File: src/content/docs/teams-and-leadership/2024 Trimester 2/leadership-t2-2024.md (hunk 2 above)
Comment (ShaunR1991) [reply src/content/docs/teams-and-leadership/2024 Trimester 2/leadership-t2-2024.md]: Updated.

**Review by ShaunR1991 — COMMENTED** [review]
Updated leadership team
Fixed formatting on team member tables
Updated the FAQ video link to the workshop to be more enduring by making use of Deakin Panopto.
Fixed conflict with PR87.

File: src/content/docs/resources/faq.mdx (hunk 1 above)
Comment (ShaunR1991) [reply src/content/docs/resources/faq.mdx]: I've fixed this.

There was a problem where the link from Teams (i.e. Sharepoint) would return a 404 or permission error.

I've migrated this video over to Panopto which should now work.

**Review by Liquidscroll — APPROVED** [review]
Great work, particularly fixing the merge conflicts!

**Review by omckeon — APPROVED** [review]
Love the new folders for the left sidebar! Particular the Resources folder :)

**Pull request merged**